pkg/translator/azure/resourcelogs_to_logs.go

```diff
@@ -207,6 +209,22 @@ func extractRawAttributes(log azureLogRecord) map[string]any {
 	return attrs
 }
 
+func copyProperties(category string, properties *any, attrs map[string]any) {
+	pmap := (*properties).(map[string]any)
+	var attrsProps map[string]any = nil
+	for k, v := range pmap {
+		if otelKey, found := ResourceLogKeyToSemConvKey(k, category); found {
+			attrs[otelKey] = v
+		} else {
+			if attrsProps == nil {
+				attrsProps = map[string]any{}
+				attrs["azure.properties"] = attrsProps
```

Comment:
What does `azure.properties` represent? Properties that don't have a semantic convention mapping?

Reply:
Sweet